### ISSUE TITLE
General SSLEngine problem while communicating with backend over https

### ISSUE THREAD

**Issue by xabolcs** [openseedbox/openseedbox#51]
[Current](https://github.com/openseedbox/openseedbox/tree/8a68a6c4ee59578dcac11f8709babf0873e50a7a) implementation of https backend communication [is not Java8 compatible](http://stackoverflow.com/questions/10258101/sslhandshakeexception-no-subject-alternative-names-present).

```
Caused by: java.security.cert.CertificateException: No subject alternative names present
```
```
~        _            _ 
~  _ __ | | __ _ _  _| |
~ | '_ \| |/ _' | || |_|
~ |  __/|_|\____|\__ (_)
~ |_|            |__/   
~
~ play! 1.4.4, https://www.playframework.com
~
~ Ctrl+C to stop
~ 
~ using java version "1.8.0_121"
```

**Comment by xabolcs** [comment]
There is a question about importing certificates programmatically at [SO](https://stackoverflow.com/q/4325263 "How to import a .cer certificate into a java keystore?"), it could be helpful.

**Comment by xabolcs** [comment]
@gregorkistler, this looks good now. Would you like to test it?

It's on the `branch-issue-51-general-ssl-problem` branch on my forks, see the referring commits.

**Comment by gregorkistler** [comment]
Seems I´ve missed the past couple of updates.
Thanks for the hint :)
Will check on Friday

**Comment by gregorkistler** [comment]
Works like a charm, thank you.

**Comment by xabolcs** [comment]
@gregorkistler, `openseedbox/client` Docker tags are fresh again!

Please update and test them, because there were [interesting](https://travis-ci.org/openseedbox/openseedbox/jobs/369439939#L1544) [errors](https://travis-ci.org/openseedbox/openseedbox/jobs/369439938#L1529) in the build logs (at ngnix build):
```
./configure: 3: [: =: unexpected operator
./configure: 8: [: =: unexpected operator
```

**Comment by gregorkistler** [comment]
@xabolcs, yes got the same warning while building the Docker image locally.
I´ve created a PR which will fix it - seems only older Nginx versions are affected.
